libafl_qemu/librasan/fuzz/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+name = "fuzz"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+publish = false
+
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+arbitrary = { version = "1.4.1" }
+asan = { path = "../asan", default-features = false, features = [
+  "dlmalloc",
+  "guest",
+  "host",
+  "libc",
+  "linux",
+  "test",
+  "tracking",
+] }
+env_logger = { version = "0.11.6" }
+libfuzzer-sys = { version = "0.4" }
```

Comment:
This should totally use libafl_libfuzzer :P

`libfuzzer-sys = { version = "*", package = "libafl_libfuzzer" }`

Eat your own dogfood